src/ui/app/jsx/cluster-list.jsx

```diff
@@ -104,7 +106,11 @@ const Cluster = React.createClass({
                                    style={progressStyle} 
                                    label={repairRun.keyspace_name}
                                    key={repairRun.id}/>
-      )
+      );
+
+      if (repairProgress === undefined || repairProgress.length == 0) {
+          repairProgress = (<ProgressBar bsStyle="success" now={0} label={"No_repairs"}/>);
+      }
```

Comment:
what's this? 🤷‍♂️

Reply:
This changes shows an empty status bar for the case where there are no active repairs.

Reply:
As discussed we can drop this change and put in a separate PR.